src/components/ListLayout.jsx

```diff
@@ -61,6 +61,94 @@ const ListLayout = ({ items, localToken }) => {
     return timeCheck;
   }
 
+  // updates isActive property of item to true if item has 2+ purchases and has been purchased within calculated estimate
+  // isActive is defaulted to false when item is added
+  items.forEach((item) => {
+    const dateOfLastTransaction =
+      item.totalPurchases > 0 ? item.purchasedDate : item.createdAt;
+    const daysSinceLastTransaction =
+      (currentTime - dateOfLastTransaction) / oneDay;
+    if (
+      item.totalPurchases > 1 &&
+      daysSinceLastTransaction < 2 * item.previousEstimate
+    ) {
+      let itemToUpdate = {
+        ...item,
+        isActive: true,
+      };
+      setUpdateToDb(localToken, item.id, itemToUpdate);
+    }
+  });
+
+  //this sorts by previousEstimate (calculation of when user will buy the item again) Note: items is already sorted alphabetically by item Id which is the normalized item name
+  items
+    .sort((itemA, itemB) => itemA.previousEstimate - itemB.previousEstimate)
+    .sort((itemA, itemB) => Number(itemB.isActive) - Number(itemA.isActive));
+  let frequencyGroup;
+  function colorClass(item) {
+    //inactive = gray
+    if (!item.isActive) {
+      frequencyGroup = 'inactive';
+      return 'bg-gray-300';
+    }
+
+    //buy soon
+    else if (item.previousEstimate < 7) {
+      frequencyGroup = 'buy soon';
+      return 'bg-green-500';
+    }
+    //kind of soon
+    else if (item.previousEstimate < 30) {
```

Comment:
it should be <  14

Reply:
`<30` seems to match what was said in the PR, but it does make sense to show the `<14` somehow since that lines up with the initial estimates.

Reply:
Yeah the acceptance criteria for this is to have groups of items < 7 days, items < 30 days , items > 30 and then the inactive which Shelley and I thought was good coverage but we can definitely brainstorm if we want to do this a different way